aduro/helpers.py

```diff
@@ -0,0 +1,25 @@
+"""Helper methods for the Seluxit API."""
+from typing import Any
+
+
+def try_convert_object(obj: Any, field_name: str) -> Any:
+    """Try convert the object to the correct value."""
+    orig_value = obj.get(field_name)
+    try:
+        if "." in orig_value:
+            value = float(orig_value)
+            return value
+        raise ValueError
+    except ValueError:
+        try:
+            value = int(orig_value)
+            return value
+        except ValueError as err:
+            try:
+                if orig_value == "true":
+                    return True
+                if orig_value == "false":
+                    return False
+                raise ValueError from err
```

Comment:
This really deep nesting of ValueErrors makes me a little dizzy. Possible to de-nest in some way?

Reply:
I was thinking aboutt it my self and I'm glad you made me change it 😃